okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/InstrumentedInterceptor.java

```diff
@@ -32,46 +34,26 @@
 
     private static final Logger log = LoggerFactory.getLogger(InstrumentedInterceptor.class);
 
-    private final Meter informational;
-    private final Meter successful;
-    private final Meter redirection;
-    private final Meter clientError;
-    private final Meter serverError;
-    private final Meter other;
+    private final LoadingCache<String, HostMetrics> hostMetrics;
 
     InstrumentedInterceptor(MetricRegistry registry, String name) {
-        informational = registry.meter(MetricRegistry.name(name, "response", "family", "informational"));
-        successful    = registry.meter(MetricRegistry.name(name, "response", "family", "successful"));
-        redirection   = registry.meter(MetricRegistry.name(name, "response", "family", "redirection"));
-        clientError   = registry.meter(MetricRegistry.name(name, "response", "family", "client-error"));
-        serverError   = registry.meter(MetricRegistry.name(name, "response", "family", "server-error"));
-        other         = registry.meter(MetricRegistry.name(name, "response", "family", "other"));
+        this.hostMetrics = CacheBuilder.newBuilder()
```

Comment:
per previous comment, would prefer to move this logic into the HostMetrics class with `record(host, code)`

Reply:
done. created HostMetricsRegistry. also moved HostMetrics and HostMetricsRegistry into a sub package